cfgov/jinja2/v1/index.html

```diff
@@ -174,15 +174,15 @@ <h3 class="h1 m-card_heading">97% of consumers</h3>
                         <section class="content-l_col
                                         content-l_col-1-3">
                             {% for block in sections.value %}
-                                <div class="m-related-posts_list-container">
+                                <div class="m-related-posts_list-container m-list">
```

Comment:
That's fine, let's leave a note here too (sorry to ask you to leave so many, I just worry we're going to lose track in the future).